### ISSUE TITLE
Intermittent problem with loading rich text boxes 

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#12159]
For a  while now I have been getting user reports of textbox type questions appearing non-editable. IIRC, the Tiny library that we use for rich text boxes is loaded from a different CDN server. If that is the case, it may due to the CDN being unavailable at time for some parts of the world, through some network paths. I usually advise the users to try again later or try using a different network, which seems to resolve the issue. 
Still, it would be good if we can avoid/minimize this intermittent issue. We can explore other options such as keeping our own copy and/or implementing a fallback mechanism (e.g., if the CDN is unreachable, load from our own copy).

**Comment by damithc** [comment]
I continues to receive occasional report of this problem.

**Comment by damithc** [comment]
More user reports about this. I think we need to fix this one way or other, as this affects a critical functionality. Loading the page using a different network may not always be possible for users.

**Comment by wkurniawan07** [comment]
> the Tiny library that we use for rich text boxes is loaded from a different CDN server

This is incorrect; the same CDN is used for Bootstrap, Handsontable, Fontawesome, and tinymce.

**Comment by samuelfangjw** [comment]
@damithc, do we know which regions are problematic.

**Comment by damithc** [comment]
> @damithc, do we know which regions are problematic.

@samuelfangjw Last week it was South Africa. Now it is France. 

> A follow up question, "textbox type questions appearing non-editable". Can I confirm the textbox is present but non-editable (disabled). The other case would be that the textbox is not present at all.

The only screenshot I have of a case like this is a student accessing from China (using VPN). In that case the text box doesn't appear at all (just a blank area). I'll ask for a screenshot when someone else reports this next time.
